operator/config/overlays/cnsa-k8s/kustomization.yaml

```diff
@@ -41,5 +41,5 @@ patches:
                   - name: CSI_RESIZER_IMAGE
                     value: cp.icr.io/cp/spectrum/scale/csi/csi-resizer@sha256:be6a7de1d43dba90710b61bd3d0d8f568654a6adadaeea9188cf4cd3554cbb87
                   - name: CSI_DRIVER_IMAGE
-                    value: cp.icr.io/cp/spectrum/scale/csi/ibm-spectrum-scale-csi-driver@sha256:fe72e11647adda33460df7ff9141f7fbd4040ea02da384a89738a9f87b4967e7
-                image: icr.io/cpopen/ibm-spectrum-scale-csi-operator@sha256:c8e9f9816ca3140304a8a290107ba5f5e3bafe62b31f58f1d3b7b81060b75ec6
+                    value: cp.icr.io/cp/spectrum/scale/csi/ibm-spectrum-scale-csi-driver@sha256:156433e8952759482a1063e46b681373467c2e7123c928a501743d079fe96f3
```

Comment:
0 is missing in this sha : it should be `0156433e8952759482a1063e46b681373467c2e7123c928a501743d079fe96f3`

Reply:
Done